test/common/services/kibana_server/version.js

```diff
@@ -10,6 +12,11 @@ export class KibanaServerVersion {
     }
 
     const status = await this.kibanaStatus.get();
+    if (status && status.version && isString(status.version)) {
```

Comment:
Can we do this without lodash?  `typeof status.version === 'string'` should be fine for our usage here, no?